public/index.php

```diff
@@ -1,5 +1,10 @@
 <?php include "templates/header.php"; ?>
 
+<?php 
+auth_loggedin();
+
+?>
+
```

Comment:
What's the idea of invoking this here?

Reply:
We have a message in the index page right now that says "You are logged in". The method being there makes sure that the user is actually logged in and that "current_user" is not just being read from the _SESSION variable.